### DISCUSSION TITLE
Is there any way to enable QuickSync VIdeo HEVC encoding?

### DISCUSSION THREAD

**Discussion by ksio89** [Xaymar/obs-stream-effects#949] [Ideas & Feedback]
I installed the latest versions of OBS Studio and StreamFX plugin, but noticed there's no support for H.265 encoding via QSV for Intel GPUs, only hevc_amf and hevc_nvenc for AMD and Nvidia cards. My laptop has a Core i5 with an Iris Xe Graphics G7 (80 EUs) and there's support for hevc_qsv on ShotCut which works great, so I'm missing it on OBSS. 

Are there any plans to add support or is there any technical issue that prevents the implementation of this encoder? Thanks in advance.

**Comment by Xaymar** [comment]
Funding prevents it.

**Reply by ksio89** [reply]
Nevermind, I enabled H.265 QSV by downloading full shared package of FFmpeg binaries and moving all .dll files to OBS Studio bin folder, and now both `libx265` and `hevc_qsv (libx265)` are available in the list of custom video encoders.

I did a quick test recording a short video with hevc_qsv, and CPU usage dropped from around 53.2% to 4.4%. Also, encoder details on MediaInfo are different than libx265, so it seems to be working fine.

**Reply by Xaymar** [reply]
Oh you were talking about Custom Output. That is not made by StreamFX.
